cookbooks/bcpc/templates/default/apache-graphite-web.conf.erb

```diff
@@ -5,9 +5,10 @@
 ################################################
 
 Listen <%="#{node[:bcpc][:graphite][:ip]}"%>:<%="#{node[:bcpc][:graphite][:web_port]}"%>
+Listen <%="#{node[:bcpc][:management][:ip]}"%>:8888
```

Comment:
Why the second `Listen` hardcoded to `8888` and why not use `node[:bcpc][:graphite][:ip]`?